code/game/objects/items/robot/items/storage.dm

```diff
@@ -190,6 +190,20 @@
 	handle_reflling(arrived)
 	return ..()
 
+/obj/item/borg/apparatus/beaker/service2
```

Comment:
This path name might confuse future contributors for its similarity with the other one. Also, it could use an autodoc.

Reply:
how exactly do ya make an autodoc

Reply:
Put a comment above a path, proc or variable, but with three slashes instead of two, or:
```
/**
  *
  *
  */
```
for multiline comments.